inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/feature/QualifierFeatureEditor.java

```diff
@@ -377,7 +377,7 @@ private void setSelectedKBItem(KBHandle value, Item<LinkWithRoleModel> aItem,
         if (linkedAnnotationFeature == null) {
             linkedAnnotationFeature = getLinkedAnnotationFeature();
         }
-        List<KBHandle> handles = new ArrayList<>();
+        List<KBHandle> handles;
```

Comment:
This is likely not correct. The method will now potentially return null, which might not be expected.